manifests/init.pp

```diff
@@ -67,8 +67,10 @@
 #       Set the value of the statsd_forward_port varable. Used to forward all
 #       statsd metrics to another host.
 #   $manage_repo
-#       Boolean to indicate whether this module should attempt to manage
-#       the package repo. Only for RPM-based distros. Default true.
+#       Deprecated. Use $manage_install instead.
```

Comment:
Should this really be deprecated? I think `$manage_repo` and `$manage_install` have two different purposes:
- `$manage_install` indicates that you want the module to install the Agent (which you can turn off if you have custom modules installing it for you)
- `$manage_repo` indicates that you want the module to set up the Agent repo (which you can turn off if you have custom modules setting it up for you).

Moreover, the current PR does not really fix #604, as it's still not possible to install the Agent without setting up the repos on Ubuntu.

Reply:
It's true that `manage_install` has more implications than `manage_repo` but since `manage_repo` wasn't implemented for Ubuntu, I thought that a wider thing like `manage_install` (which we needed anyway for #476) could also work for both use cases.

Maybe the wording of the deprecation message should be `Set manage_install to false and install datadog-agent manually` to achieve the same thing as before.

Another option would be to keep `manage_repo` but then we have to implement it for Ubuntu as well and given how little these options are used, I prefer having one flag that two.